app/controllers/sales_controller.rb

```diff
@@ -0,0 +1,39 @@
+class SalesController < ApplicationController
```

Comment:
Examine the indentation here in this file as well, looks like some common copy/paste issues